packages/app/src/sandbox/eval/tests/jest-lite.js

```diff
@@ -102,12 +116,17 @@ export default class TestRunner {
       });
     };
 
+    const { JSDOM } = JSDOM2;
+    const { window } = new JSDOM('');
```

Comment:
This should maybe be initialized the same way it is in jest?

https://github.com/facebook/jest/blob/4d3c1a187bd429fd8611f6b0f19e4aa486fa2a85/packages/jest-environment-jsdom/src/index.ts#L33-L39

Reply:
Ah nice, thanks for the pointer! I'll take a look, maybe we can have a 1:1 implementation!